notebooks/asian_barrier_option/elu_activation/plugins/eluPlugin.cu

```diff
@@ -0,0 +1,296 @@
+/*
+ * Copyright (c) 2019, NVIDIA CORPORATION.  All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <cassert>
+#include <cstring>
+#include <vector>
+
+#include "NvInfer.h"
+#include "eluPlugin.h"
+#include "pluginKernels.h"
+#include "common.h"
+#include "logger.h"
+
+using namespace nvinfer1;
+
+namespace bert
+{
+
+namespace test
+{
```

Comment:
To clean up I would rename `bert` to `elu` and remove `test` namespace in file "eluPlugin.cu". Do the same in files: "eluPlugin.h", "pluginKernels.h", and "pluginUtil.h".

Reply:
done